seqr/views/apis/anvil_workspace_api.py

```diff
@@ -240,3 +268,89 @@ def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
     )
 
     safe_post_to_slack(SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, message_content)
+
+
+def generate_AnVIL_variables(project, data_path, sample_type):
+    pipeline_dag = {
+        "active_projects": [project.guid],
+        "es_enable_export": 'true',
+        "vcf_path": data_path,
+        "project_path": '{}v1'.format(_get_loading_project_path(project, sample_type)),
+        "projects_to_run": [project.guid],
+    }
+    return pipeline_dag
+
+def get_variables(variable_key):
```

Comment:
the word "variables" usually refers to variables in the code, so the name of this function is a little confusing. Maybe something like `_get_dag_variables`